Cargo.toml

```diff
@@ -10,20 +10,20 @@ license = "MIT/Apache-2.0"
 name = "slack-hook2"
 readme = "README.md"
 repository = "https://github.com/clux/rust-slack"
-version = "0.10.0"
+version = "0.10.1"
 edition = "2018"
 
 [dependencies]
-chrono = "0.4"
-hex = "0.4"
-serde = "1"
-serde_derive = "1"
-serde_json = "1"
-url = { version = "2", features = ["serde"] }
-thiserror = "1.0.11"
+chrono = "^0.4"
+hex = "^0.4"
+serde = "^1"
+serde_derive = "^1"
+serde_json = "^1"
+url = { version = "^2", features = ["serde"] }
+thiserror = "^1.0"
 
 [dependencies.reqwest]
-version = "0.10.1"
+version = "~0.11.12"
```

Comment:
reqwest tends to break from one version to another so less freedom in the version.